test/connectors/cache/FileUploadCacheConnectorSpec.scala

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors.cache
+
+import com.github.tomakehurst.wiremock.client.WireMock._
+import connectors.Reference
+import models.{FileUploadStatus, InProgress, UploadId, UploadedSuccessfully}
+import org.scalatest._
+import org.scalatest.matchers.must.Matchers
```

Comment:
Should this file not be UploadProgressTrackerSpec since this is what it appears to be testing?